test/test_helper.rb

```diff
@@ -6,12 +6,7 @@
 
 require 'active_support/all'
 
-require 'xirr/config.rb'
-require 'xirr/base.rb'
-require 'xirr/bisection.rb'
-require 'xirr/newton_method.rb'
-require 'xirr/cashflow.rb'
-require 'xirr/transaction.rb'
+require 'xirr'
 include Xirr
```

Comment:
It would be nice to remove this global inclusion too. It could potentially make code works only in specs and blow up elsewhere.

Reply:
@radarek Cleaned up